plutus-ledger/src/Ledger/Index.hs

```diff
@@ -395,10 +393,11 @@ checkTransactionFee tx =
 mkPV1TxInfo :: ValidationMonad m => Tx -> m PV1.TxInfo
 mkPV1TxInfo tx = do
     slotCfg <- pSlotConfig . vctxParams <$> ask
-    txins <- traverse mkPV1TxInInfo $ view inputs tx
+    txInputs <- traverse mkPV1TxInInfo $ view inputs tx
+    let txInfoOutputs = map (\(TxOut txOut) -> fromCardanoTxOut txOut) $ txOutputs tx
```

Comment:
The type is simply `TxOut`, so the variable name should be simply `plutusTxOutputs`.